demos/image_generation/README.md

```diff
@@ -117,17 +117,21 @@ Image generation endpoints consist of 3 steps: text encoding, denoising and vae
 :::{tab-item} Docker (Linux)
 :sync: docker
 In case you want to use Intel NPU device to run the generation, add extra docker parameters `--device /dev/accel --group-add=$(stat -c "%g" /dev/dri/render* | head -n 1)` to `docker run` command, use the docker image with NPU support. Export the models with precision matching the NPU capacity and adjust pipeline configuration.
+In this specific case, we also need to use `--device /dev/dri`, because we also use GPU.
 
 > **NOTE:** The NPU device requires the pipeline to be reshaped to static shape, this is why the `--resolution` parameter is used to define the input resolution.
 
+> **NOTE:** This feature will be available in 2025.3 and later releases, so until next release, it is required to build the model server from source from the `main` branch.
+
+
 It can be applied using the commands below:
 ```bash
 mkdir -p models
 
 docker run -d --rm -p 8000:8000 -v $(pwd)/models:/models/:rw \
-  --user $(id -u):$(id -g) --device /dev/accel --group-add=$(stat -c "%g" /dev/dri/render* | head -n 1) \
+  --user $(id -u):$(id -g) --device /dev/accel --device /dev/dri --group-add=$(stat -c "%g" /dev/dri/render* | head -n 1) \
   -e http_proxy=$http_proxy -e https_proxy=$https_proxy -e no_proxy=$no_proxy \
-  openvino/model_server:2025.2-gpu \
+  openvino/model_server:2025.3-gpu \
```

Comment:
Not sure if we should use this tag anyway. Until it's not released, maybe we could use the default tag that model server image has when built with `make docker_build`. What do you think?